config/hyde.php

```diff
@@ -284,7 +284,7 @@
     */
 
     'authors' => [
-        Author::create(
+        'mr_hyde' => Author::create(
             'mr_hyde', // Required username
```

Comment:
Need an elegant way to remove the duplication of username entries here, also need to figure out how to handle discrepancies between them

Reply:
https://github.com/hydephp/develop/issues/1797